resources/gemini-1b/provisioner.tf

```diff
@@ -28,8 +28,8 @@ resource "null_resource" "setup_nodes" {
     host = digitalocean_droplet.gemini-1b[count.index].ipv4_address
     user = "root"
     type = "ssh"
-    agent = true
-    agent_identity = var.ssh_identity
```

Comment:
why do you want to remove the identity?

Reply:
I haven't found another way to connect cloud terraform "null_resource" to droplets.
In case if you know any ways to do it - please share.